lib/mayaUsd/ufe/UsdUndoReorderCommand.cpp

```diff
@@ -28,6 +30,11 @@ UsdUndoReorderCommand::UsdUndoReorderCommand(
     , _parentPrim(parentPrim)
     , _orderedTokens(tokenList)
 {
+    // Apply restriction rules
+    for (const auto& childPrim : parentPrim.GetChildren()) {
+        ufe::applyCommandRestriction(childPrim, "reorder");
+        break;
+    }
```

Comment:
@ppt-adsk  this change is part of the fix for internal MAYA-106089.

Reply:
@ppt-adsk  Added UFE guard to protect the code against PR120. I will clean up the temporary guards once PR121 is out.